Source/WebCore/page/ResizeObserver.cpp

```diff
@@ -148,17 +162,24 @@ void ResizeObserver::deliverObservations()
 
     auto targetsWaitingForFirstObservation = std::exchange(m_targetsWaitingForFirstObservation, { });
 
+    if (isNativeCallback()) {
+        std::get<NativeResizeObserverCallback>(m_JSOrNativeCallback)(entries, this);
```

Comment:
`(entries, *this)` would be closer to the JS case

Reply:
done